src/main/java/dinkplugin/message/NotificationBody.java

```diff
@@ -30,6 +31,9 @@
     T extra;
     @NotNull
     transient String text;
+    @Nullable
+    @EqualsAndHashCode.Exclude
+    transient String thumbnailUrl;
```

Comment:
I'd argue this is a Discord field rather than a Dink field

A small comment might be nice too

Reply:
Updated in https://github.com/pajlads/DinkPlugin/pull/119/commits/e032bd01a11f83c659c2b3bf07817a22de4cf44b